src/coprocessor/endpoint.rs

```diff
@@ -276,7 +280,8 @@ impl Host {
                         let mut exec_metrics = ExecutorMetrics::default();
                         ctx.collect_metrics_into(&mut exec_metrics);
                         tracker.record_handle(Some(&mut resp), exec_metrics);
-                        future::ok::<_, ()>(on_resp.respond(resp))
+                        on_resp.respond(resp);
+                        future::ok::<_, ()>(())
```

Comment:
what's the difference with the previous implement?

Reply:
they are the same. It's for keeping style consistent with other `CopRequest`.